### ISSUE TITLE
Add info on how to launch with mockup network

### ISSUE THREAD

**Issue by jgvictores** [asrob-uc3m/robotDevastation#62]
- [ ] Make sure nothing breaks if launch with mockup network.
- [ ] Add info on how to launch with mockup network.

**Comment by David-Estevez** [comment]
mockup network == MockupNetworkManager ?

**Comment by jgvictores** [comment]
Yes. The basic idea is to be able to try without launching an rdServer. 
Right now we do not know what coukd happen if we use this mock object.
